.env

```diff
@@ -100,6 +100,9 @@ DISCOVER_API_KEY=
 # OrderCloud
 ORDERCLOUD_BUYER_CLIENT_ID=
 ORDERCLOUD_BASE_API_URL=
+ORDERCLOUD_MIDDLEWARE_CLIENT_ID=
+ORDERCLOUD_MIDDLEWARE_CLIENT_SECRET=
+ORDERCLOUD_WEBHOOK_HASH_KEY=
```

Comment:
This environment variable does not seem to be used. Could it be removed?

Reply:
Its actually used by the [ordercloud javascript catalyst](https://github.com/ordercloud-api/ordercloud-javascript-catalyst/blob/main/README.md#usage) library that Oliver built. It is used to authenticate our webhook endpoints and setting it as an environment variable allows that to be done globally.

I could add a comment to make that more clear, what do you think?

Reply:
No need for a comment in this file. Just in the `docker-compose.override.yml`. Thanks